### ISSUE TITLE
Cannot access More Info from Guided Tour using VoiceOver

### ISSUE THREAD

**Issue by steinbro** [soundscape-community/soundscape#100]
> I can access the BHA Grounds route, I can not access the audio or image files virtually from within the route. This occurred previously when I attempted to access this info virtually. Today when I was walking the route with Emily she helped me access this information, but when I attempt to access this information virtually, the audio files and images subtabs are not announced after selecting More Info as expected.

From @fraziercarr:
> For the Guide Tour accessibility, I haven’t found a point with the audio files, but I can see an issue with Voiceover where I can’t always get to the images. Voiceover appears to get stuck in a loop on the map and there you cannot get past the map and to the images. It doesn’t happen all the time though. This one feels like it could be an iOS bug to me given that it’s only happening in the map section.

**Comment by jchudge** [comment]
Following reported by a user; and it might be related to this bug!

 The "Call Out Beacon" action in a route waypoint doesn't say anything with Voiceover. Double-tapping "Call Out Beacon" does not respond. In contrast, "Call Out Beacon" works correctly when setting up a single beacon.
